### ISSUE TITLE
@ngx-loading-bar 1.1.0 throws error "XMLHttpRequest is not defined" on plattform-server

### ISSUE THREAD

**Issue by denisyilmaz** [aitboudad/ngx-loading-bar#51]
I updated ngx-loading-bar to latest version. unfortunately, on plattform-server (universal production build) I get following error:
`ReferenceError: XMLHttpRequest is not defined
    at BrowserXhr.build …`

Even when checking if platform is browser `<ngx-loading-bar *ngIf="isPlatformBrowser"></ngx-loading-bar>`(to prevent rendering of loading bar on server-side) I get the error.

Angular Version 5.2.0

**Comment by aitboudad** [comment]
hi, just tested on my side using https://github.com/angular/universal-starter and seems everything work as expected.
Please ensure that `LoadingBarHttpClientModule` is registred before [ServerModule](https://github.com/angular/universal-starter/blob/master/src/app/app.server.module.ts#L13).

Thanks!

**Comment by denisyilmaz** [comment]
ok, I figured it out.
I was adding the `<ngx-loading-bar></ngx-loading-bar>` for styling purpose in an sub-module, not the AppModule. This would normally work fine, but this time I made this sub-module lazy loaded, what results in the error mentioned above. Don't know if that's a bug or just limitations with lazy loaded modules.

Moving `<ngx-loading-bar></ngx-loading-bar>` in the main `AppModule` solved the problem.
